packages/apps/workshop/stories/design-system/components/button/button-webcomponent-primary.stories.js

```diff
@@ -1,14 +1,14 @@
 import { boolean } from '@storybook/addon-knobs';
 import { forModule } from 'storybook-addon-angularjs';
 
-import Button from '@ovh-ux/ui-kit.button';
+import ouiButton from '@ovh-ux/ui-kit/dist/js/oui-button';
```

Comment:
To keep consistency with other stories
```suggestion
import '@ovh-ux/ui-kit/dist/js/oui-button';
```